# Baxter IR, Ziegler G, Lahner B, Mickelbart MV, Foley R, Danku J, et al. (2014) Single-Kernel Ionomic Profiles Are Highly Heritable Indicators of Genetic and Environmental Influences on Elemental Accumulation in Maize Grain (Zea mays). PLoS ONE 9(1): e87628. <https://doi.org/10.1371/journal.pone.0087628>

## Abstract: The ionome, or elemental profile, of a maize kernel can be viewed in at least two distinct ways. First, the collection of elements within the kernel are food and feed for people and animals. Second, the ionome of the kernel represents a developmental end point that can summarize the life history of a plant, combining genetic programs and environmental interactions. We assert that single-kernel-based phenotyping of the ionome is an effective method of analysis, as it represents a reasonable compromise between precision, efficiency, and power. Here, we evaluate potential pitfalls of this sampling strategy using several field-grown maize sample sets. We demonstrate that there is enough genetically determined diversity in accumulation of many of the elements assayed to overcome potential artifacts. Further, we demonstrate that environmental signals are detectable through their influence on the kernel ionome. We conclude that using single kernels as the sampling unit is a valid approach for understanding genetic and environmental effects on the maize kernel ionome.

## TL;DR : The authors used this study as a proof-of-concept for single kernel ICP-MS testing to analyze the ionome of the maize plant. They dissected this question by asking first where areas of confounding could occur, and proceeded to answer 1) Is single kernel ICP-MS testing confounded by genotypic differences, the environment, or the GxE interaction? 2) Does the kernel location on the cob confound the ICP-MS results? 3) Does the organic composition (sugary lines of corn) confound the ICP-MS testing? 4) Is the single kernel ICP-MS approach robust for QTL studies?

## Figure 1 - Mg25 & Cd111 Scatter Plot

The first figure is a visualization of the mean concentrations for two (Mg25 and Cd111) of the twenty elements for 23 of the 26 genotypes across the 2010 and 2011 sampling years. This figure is a way to visualize the significant correlation in element concentration across the two sampling years.

We had the raw data for the paper, so I first had to remove outlier data using a outlier-removal function. Once the outliers were removed, I trimmed the data to only include the data I needed for the plot: element concentration, genotype, and sampling year. I now had a df with four to six of each genotype each with their own element concentrations. I wrote a code to calculate the mean of each genotype for each element and each sampling year. This yielded a df with the 26 genotypes having a mean value for each element and for each sampling year (2 columns per genotype). I next removed the genotypes that were excluded from the data (popcorn and sweet corn genotypes) and began to generate plots for Mg25 and Cd111. To generate the plots, I created a df for each element that contained the concentrations for each genotype and both years. I stored these dfs in a list so I could extract whichever element I wanted to plot by just specifying the element name. I then created a scatter plot for both Mg25 and Cd111 and created a plot grid of 2 to combine the two plots. 

## Table 4 turned into visualization

### The paper presents QTL findings in a tabular format and I decided to replicate the data for chromosome 1 and plot a visualization. I chose sulfur to replicate on chromosome 1 because it had the largest effect on that chromosome. I chose chromosome 1 because I am a simple person and 1 is before the other numbers. To generate files for this analysis, I created a map file with chromosome, marker, and allelic information for the lines, and the elemental phenotypes for each line. Then, I needed to merge the two files to create a file useful for QTL mapping. However, I did not have marker positions. I found a thesis that used what seemed to be the same set of markers as the paper and manually entered them using a diagram. I feel there was a lot of error that could happen during this step. Using the file that was imputed with marker positions, I used R/Qtl package to do 1) QTL scan of chromosome 1, 2) QTL effect from the marker found on the chromosome scan, 3) scanning for other potential markers in the QTL window.

#### 1) QTL scan: I did have a peak near the QTL reported in the paper, however, upon further examination I was about 2 cM off

#### 2) QTL effect: I contrasted the QTL effect of the marker reported in the paper and the marker presented from the QTL scan. The marker I found in step 1) did not have any heterogeneity and looks horrible. I would not trust these results. To me, it looks like there is a huge issue with the allelic information or the manually entered marker positions

#### 3) Other potential markers: The window for other potential markers nearest the QTL is promising in the fact that they are relatively close to one another, but the marker from the paper is not within the potential QTL.

### Table 4 TL;DR: The pictures look nice and near the QTL, but I think there are large issues from not having the marker positions. I think it may be coincidence that the marker I found is near the QTL marker reported in the paper. For annotated scripts and an additional ReadMe for Table/Figures 4, please look under the `Code` folder.
